IdentityCore/src/cache/metadata/accountMetadata/MSIDAccountMetadataCacheKey.m

```diff
@@ -24,14 +24,14 @@
 #import "MSIDAccountMetadataCacheKey.h"
 #import "MSIDCacheKey.h"
 
+static NSString *MSID_APP_METADATA_ACCOUNT_KEY_VALUE = @"msid_account_metadata_account_key";
 static NSInteger kAuthorityMapMetadataType = 5001;
 
 @implementation MSIDAccountMetadataCacheKey
 
-- (instancetype)initWitHomeAccountId:(NSString *)homeAccountId
-                            clientId:(NSString *)clientId
+- (instancetype)initWithClientId:(NSString *)clientId
 {
-    self = [super initWithAccount:homeAccountId ?: nil
+    self = [super initWithAccount:MSID_APP_METADATA_ACCOUNT_KEY_VALUE
```

Comment:
nit: should we just use MSID_APP_METADATA_AUTHORITY_MAP_TYPE as account, and clientId as service?

Reply:
changed :)